Header header
float32 Period
Gains Kp
Gains Kd
Gains Ki
